Sources/SwiftDocC/Model/Rendering/RenderNodeTranslator.swift

```diff
@@ -1253,12 +1253,6 @@ public struct RenderNodeTranslator: SemanticVisitor {
             
             return availability.availability
                 .compactMap { availability -> AvailabilityRenderItem? in
-                    // Filter items with insufficient availability data.
-                    // Allow availability without version information, but only if
-                    // both, introduced and deprecated, are nil.
-                    if availability.introducedVersion == nil && availability.deprecatedVersion != nil {
```

Comment:
Question: is it possible to write `@available` attributes without either an introduced or deprecated version. If so, what do we want to do with such availability items? Do we want to display them or filter them out?

In other words, should we remove this check completely (to display such availability items) or should we update the check to see that some version information is non-nil:
```swift
if availability.introducedVersion == nil, 
   availability.deprecatedVersion == nil, 
   availability.obsoletedVersion  == nil
{
   return nil
}
```

Reply:
Good catch, the `@available` attribute accepts the following values:
`Expected 'available' option such as 'unavailable', 'introduced', 'deprecated', 'obsoleted', 'message', or 'renamed'`.

`unavailable` `introduced` and `deprecated` are correctly handled by DocC.
`message` and `renamed` don't surface the symbol graph.

The only case left it's `obsoleted`, this is not something we represent in the documentation today so in this commit `https://github.com/swiftlang/swift-docc/pull/1079/commits/d07cf9a0c7ef7eb1305da3acfca1adeb858f8936` I'm filtering it out.